src/clients/api/mutations/vote/useCastVote.ts

```diff
@@ -18,7 +19,28 @@ const useCastVote = (
         fromAccountAddress,
         ...params,
       }),
-    options,
+    {
+      ...options,
+      onSuccess: (...onSuccessParams) => {
+        const { proposalId, voteType } = onSuccessParams[1];
+
+        // Invalidate query to fetch voters
+        queryClient.invalidateQueries([
```

Comment:
We likely want to invalidate this endpoint the same way so that is isn't only updated when the user votes but updated with ever block so all votes are updated.